### PR TITLE
Projectiondefaults, width slider direct save

### PR DESCRIPTION
**Pull request by FinnStutzenstein**
The projection default as a root rest element is removed. This reduces the complexity on the client a lot.

### CODE REVIEW

File: client/src/app/core/repositories/projector/projection-default-repository.service.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+     * @param mapperService Maps collection strings to classes
+     * @param dataSend sending changed objects
+     * @param http
+     */
+    public constructor(
+        DS: DataStoreService,
+        dataSend: DataSendService,
+        mapperService: CollectionStringMapperService,
+        viewModelStoreService: ViewModelStoreService,
+        translate: TranslateService
```
Comment (tsiegleauq) [inline client/src/app/core/repositories/projector/projection-default-repository.service.ts]: doc does not match constructor

**Comment by normanjaeckel** [comment]
Hm. Why do you added views to delete projection defaults or to create new ones?

**Comment by normanjaeckel** [comment]
Ping?

**Comment by FinnStutzenstein** [comment]
You are right. I also disallowed update and partial updates, becuase they are not needed. See #4644 .

**Pull request merged**